packages/theme/example/index.html

```diff
@@ -1629,6 +1640,22 @@ <h4 class="modal-title">Source Code</h4>
     <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
     <script>
     (function(){
+        function setVisibilyDropDownMenu(){
+          var top = $(document).scrollTop();
+          var elementTop = $("#banner").offset().top;
+          var elementBottom = elementTop + $("#banner").height();
+
+          if ((top < elementTop) || (top < elementBottom)) {
```

Comment:
`top < elementBottom` should be sufficient.

Reply:
You can use [toggle fn](http://api.jquery.com/toggle/#toggle-display) instead like:

```javascript
$('#menu_dropdown').toggle(top < elementBottom);
```